packages/create-request-form/src/utils/invoiceTotals.ts

```diff
@@ -0,0 +1,34 @@
+export const calculateInvoiceTotals = (data: InvoiceItem[]) => {
```

Comment:
consider using getInvoiceTotal from `data-format` package.
https://github.com/RequestNetwork/requestNetwork/blob/master/packages/data-format/src/format/rnf_invoice/utils.ts#L4

Reply:
Decided to use custom utils for now.